embeddings/embedding/static/fasttext.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABC
+from dataclasses import dataclass, field
+from typing import Any, Literal, Optional
+
+from flair.embeddings import FastTextEmbeddings
+
+from embeddings.embedding.flair_embedding import FlairEmbedding
+from embeddings.embedding.static.config import SingleFileConfig, StaticModelHubConfig
+from embeddings.embedding.static.word import StaticWordEmbedding
+
+
+@dataclass
+class KGR10FastTextConfig(SingleFileConfig):
+    method: Optional[Literal["cbow", "skipgram"]] = None
```

Comment:
In my opinion it would be overkill, we do not develop library for fasttext. There is plenty of such enums in an every ml library. As long as we don't use it in many places, a string is satisfactory.